base_delivery_carrier_label/models/stock_picking.py

```diff
@@ -106,6 +107,7 @@ def action_generate_carrier_label(self):
             shipping_labels = pick.generate_shipping_labels()
             for label in shipping_labels:
                 data = pick.get_shipping_label_values(label)
+                data['datas'] = base64.b64encode(data['datas'])
```

Comment:
I don't get why you encode in base64 back when we just removed it some days ago?!
Usually carrier already return the label in base64, so base_delivery_carrier_label now expect the label data to be in base64 format already. 
You shoud encode the label directly in the module delivery_carrier_label_default.
In the method `generate_default_label` on `stock.picking`

Else it seems good to me

Reply:
yep, you are correct I missed this, updated